### PR TITLE
Added Data Modeling/Parsing Rules Instructions to Event Collector

### PR DESCRIPTION
**Pull request by kgal-pan**
## Status
Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5880

## Description
Added Data Modeling Rules Instructions to Event Collector

### CODE REVIEW

File: docs/integrations/collectors.md [hunk 1]
```diff
@@ -67,17 +67,207 @@ def main():
...
+  tags: MyVendor, Events
+  ```
+
+- `MyVendorEventCollector_1_3_schema.json`: This file contains the fields that came directly from the raw response and were used in the development of the XDM rules. The file has the following structure:
+
+  ```json
+  {
+    "MyVendor_MyProduct_raw": {
+      "field_1": {
+        "type": "string|int",
```
[the remark this answers is not in the document]
Comment (kgal-pan) [reply docs/integrations/collectors.md]: Found out there's also a `datetime` type.

File: docs/integrations/collectors.md [hunk 2]
```diff
@@ -27,17 +28,51 @@ marketplaces:
 - marketplacev2
 ```
 
+## Collect Section Parameters
+
+In order to organize all collection-related parameters in one section, we can add the following key to each relevant parameter in the integration YAML:
+
+```yaml
+configuration:
+  # ...
```
Comment (daryakoval) [inline docs/integrations/collectors.md:37]: We should add this one to the yml as well: 

```suggestion
sectionOrder:
- Connect
- Collect
configuration:
  # ...
```

**Review by daryakoval — COMMENTED** [review]
Amazing work!!! Just one addition :)

**Comment by kgal-pan** [comment]
@julieschwartz18 - FYI, these are changes we will incorporate into the XSIAM dev docs in the _Integrations and Scripts_ section.

**Pull request closed without merging**